src/components/Nft/NftCollectionRow.tsx

```diff
@@ -41,7 +42,7 @@ function NftCollectionRow({ collection, onCollectionPress }: Props) {
               ellipsizeMode="tail"
               numberOfLines={2}
             >
-              {metadata?.tokenName || collection.contract}
+              {metadata?.tokenName || nft.contract}
```

Comment:
No need for a safety `nft?.contract` here?

Reply:
Done 👌